codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -79,25 +80,15 @@ public void onStreamActive(Http2Stream stream) {
 
     @Override
     public void initialWindowSize(int newWindowSize) throws Http2Exception {
-        int delta = newWindowSize - initialWindowSize;
+        final int delta = newWindowSize - initialWindowSize;
```

Comment:
I can revert.

I had an anonymous class at one point using this....but needed to get something from it so it is no longer anonymous.